### PR TITLE
Fix label display on new issues

### PR DESCRIPTION
**Pull request by silverwind**
PR #13570 broke label rendering for new issues and pulls because I missed the fact that the code was relying on the DOM elements being toggled by JavaScript.

On top of that, the label rendering for new issues and pull was using an outdated template which I consolidated in a new shared template.

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

**Comment by silverwind** [comment]
One more adjustment pushed. Removal of wrapper element and 1px more margin between labels:

image

File: templates/repo/issue/new_form.tmpl [hunk 1]
```diff
@@ -73,14 +73,10 @@
 			<div class="ui labels list">
```
Comment (mrsdizzie) [inline templates/repo/issue/new_form.tmpl]: Could all of this code go in shared template if it is the same in both places? Everything inside 
```<div class="ui labels list">```

Comment (silverwind) [reply templates/repo/issue/new_form.tmpl]: You're right, I missed the fact that this parent matches in both.

Comment (silverwind) [reply templates/repo/issue/new_form.tmpl]: Done

**Review by mrsdizzie — APPROVED** [review]

**Comment by lafriks** [comment]
🚀

**Pull request merged**